File: BankingTool.Model/Dto/RequestDto/GetBankDetailWithoutCustomerAndAccountTypeRequest.cs
namespace BankingTool.Model.Dto.RequestDtos;
public class GetBankDetailWithoutCustomerAndAccountTypeRequestObject
{
    public GetBankDetailWithoutCustomerAndAccountTypeRequest Request { get; set; }
}
public class GetBankDetailWithoutCustomerAndAccountTypeRequest
{
    public GetBankDetailWithoutCustomerAndAccountTypeRequestCustomer Customer { get; set; }
    public GetBankDetailWithoutCustomerAndAccountTypeRequestAccount Account { get; set; }
}
public class GetBankDetailWithoutCustomerAndAccountTypeRequestCustomer : RequestId
{

}
public class GetBankDetailWithoutCustomerAndAccountTypeRequestAccount
{
    public int AccountTypeId { get; set; }
}

File: bankingtool.client/src/app/dto/request/bank-account/get-bank-detail-without-customer-and-account-type-request.ts
import { RequestId } from "../request-id";

export interface GetBankDetailWithoutCustomerAndAccountTypeRequestObject {
  request: GetBankDetailWithoutCustomerAndAccountTypeRequest;
}

export interface GetBankDetailWithoutCustomerAndAccountTypeRequest {
  customer: GetBankDetailWithoutCustomerAndAccountTypeRequestCustomer;
  account: GetBankDetailWithoutCustomerAndAccountTypeRequestAccount;
}

export interface GetBankDetailWithoutCustomerAndAccountTypeRequestCustomer extends RequestId {

}

export interface GetBankDetailWithoutCustomerAndAccountTypeRequestAccount {
  accountTypeId: number;
}
